modules/virt-deleting-virt-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * virt/install/uninstalling-virt.adoc
+
+[id="virt-deleting-virt-cli_{context}"]
+= Deleting the {VirtProductName} using the CLI
+
+You can delete {VirtProductName} by using the CLI.
+
+.Prerequisites
+
+- `oc` command installed on workstation.
+
+- Access to an {product-title} cluster using an account with
+`cluster-admin` permissions.
+
+.Procedure
+
+. Delete the HyperConverged Custom Resource:
++
+----
+$ oc delete HyperConverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Delete the subscription:
++
+----
+$ oc delete subscription kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Extract the ClusterServiceVersion (CSV) name for  {VirtProductName}:
++
+----
+$ CSV_NAME=$(oc get csv -n openshift-cnv -o=custom-columns=:metadata.name)
+----
+. Delete the CSV from the cluster by using the CSV name:
++
+----
+$ oc delete csv ${CSV_NAME} -n openshift-cnv
+----
```

Comment:
If you can, please add a confirmation step. It's weird that the CLI and web console versions seem to delete different components.

Reply:
Upon applying the quotes command in the terminal, you get (as an example for CNV version 2.4):
`clusterserviceversion.operators.coreos.com "kubevirt-hyperconverged-operator.v2.4.0" deleted`
as a response.
That can be used as a confirmation that the CSV has been deleted from the cluster, as opposed to a response like:
`Error from server (NotFound)...`

Reply:
Added a confirmation step with explanation.